src/__Test__/calculate.test.js

```diff
@@ -0,0 +1,9 @@
+import calculate from '../logic/calculate';
+
+describe('calculate', () => {
+  test('check AC button', () => {
+    const state = { total: 0, operation: null, next: null };
+    const buttonName = 'AC';
+    expect(calculate(state, buttonName)).toEqual({ total: 0, operation: null, next: null });
+  });
+});
```

Comment:
- Good job adding a test 👍 👍 , according to the project requirements you are to test all the scenarios in the `calculate.test` module, please add more scenarios to your test 🙏